internal/auth/mock/auth.go

```diff
@@ -5,34 +5,49 @@
 package mock
 
 import (
-	gomock "github.com/golang/mock/gomock"
 	reflect "reflect"
+
+	gomock "github.com/golang/mock/gomock"
 )
 
-// MockSecretStore is a mock of SecretStore interface
```

Comment:
nit: isn't this file auto generated? why the tiny changes? If it was part of a "format-on-save" action, I'd suggest reverting it because it will cause unnecessary changes once we generate mocks again

Reply:
Reverted in https://github.com/auth0/auth0-cli/pull/316/commits/6d0b3efcfe9fe399a43b93dc202291e3ed6496e6.

Reply:
thanks 🙇🏼